docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,390 @@
+
+`docker run -env PORT0=8000 -p 80:8000 -t docker-user-name/time-server:part1`
+
+Note that we are reading the port number from the `PORT0` environment variable. Marathon sets this environment variable when launching the container and since we don't have that yet, we need to provide the environment variable manually using `-env PORT0=8000`. The `-p` option maps the host port 80 to the container port 8000. The  `-t` flag creates a pseudoTTY and since we unbuffered the Python standard I/O in our Dockerfile, we will be able to see the real time logs of the server in the console. Once you executed the above command, you should be able to browse [localhost](http://localhost:80).
+
+
+#### Tag and publish your container
+Once you are satisfied with the functionality of your container, you can publish the Docker image on to the Docker registry. In our case, we execute :
+
+`docker tag time-server docker-user-name/time-server:part1`
+
+This tags our `time-server` image with the Docker repository `time-server` in your Docker user name `docker-user-name` and provides an optional tag `part1`.
+
+Once we tag our image, we have to publish (synonmous with github push) to the Docker registry so that Marathon would be able to discover this in future using an url. We achieve this by executing the command :
+
+`docker push docker-user-name/time-server:part1`
+
+Now that we have the container ready, in the next section we will see how to create a package!
+
+### Step 3 : Creating a DC/OS Package
+In order to create a package, you need to have forked the [Universe repo](https://github.com/mesosphere/universe) and then cloned it so that it is available in your terminal. Once you do this, create a directory named time-server under the repo/package/T directory (as our package name starts with the letter "t"). Inside this directory, if there is already another package with a name of your choice, you have to name your package differently. We create all the required files in this package.
+
+After you read this step, if you need further examples, you can refer to the [repo/packages/H/hello-world](repo/packages/H/hello-world) package.
+
+Each package has its own directory, with one subdirectory for each package revision. Each package revision directory contains the set of files necessary to create a consumable package that can be used by a DC/OS Cluster to install the package. For example, our package would look like this:
+
+```
+└── repo/package/T/time-server
+    ├── 0
+    │   ├── config.json
+    │   ├── resource.json
+    │   ├── marathon.json.mustache
+    │   └── package.json
+    └── ...
+```
+
+In our case, since this is the first version of our time-server, we will create the above directory structure with only one revision (with number 0) and create the required empty files. As the versions of your package grow, this number increments by one unit.
```

Comment:
Would this be a good place to explain that once a revision has been added to Universe, its files should never be modified? A new revision must be created for any change.